opaque_keys/edx/tests/test_locators.py

```diff
@@ -40,7 +40,7 @@ def test_description_locator_url(self):
     def test_description_locator_version(self):
         object_id = '{:024x}'.format(random.randrange(16 ** 24))
         definition_locator = DefinitionLocator('html', object_id)
-        self.assertEqual(object_id, str(definition_locator.version()))
+        self.assertEqual(object_id, str(definition_locator.version))
```

Comment:
Why you made this change ?

Reply:
quality failure 
`definition_locator.version is not callable (not-callable)`
we are accessing a property here.